apps/aecore/src/aec_peers.erl

```diff
@@ -675,3 +677,23 @@ ping_interval() ->
 
 sync_port() ->
     aeu_env:user_config_or_env([<<"sync">>, <<"port">>], aecore, sync_port, ?DEFAULT_SYNC_PORT).
+
+parse_peer_address(PeerAddress) ->
+    case http_uri:parse(PeerAddress) of
```

Comment:
I am trying to understand if this is used for parsing unsanitized URIs.

Reply:
It is used to parse whatever the user writes in his configuration

Reply:
I see - the peers in the P2P messages are still exchanged as distinct elements (i.e. not as URI). Ok.

Reply:
Yes exactly!